prefect_utils/mysql.py

```diff
@@ -0,0 +1,103 @@
+"""
+Tasks for interacting with Aurora MySQL.
+"""
+import mysql.connector
+from prefect import task
+from prefect.engine import signals
+from prefect.utilities.logging import get_logger
+
+
+@task
+def load_s3_data_to_mysql(
+    aurora_credentials: dict,
+    database: str,
+    table: str,
+    table_schema: list,
+    s3_url: str,
+    field_delimiter: str = ',',
+    enclosed_by: str = '',
+    escaped_by: str = '\\\\',
+    overwrite: bool = False,
+):
+
+    """
+    Loads an Aurora MySQL database table from text files in S3.
+
+    Args:
+      aurora_credentials (dict): Aurora credentials dict containing `user`, `password` and `host`.
+      database (str): Name of the destination database.
+      table (str): Name of the destination table.
+      table_schema (list): List of tuples specifying table schema.
+              Example: `[('id', 'int'), ('course_id', 'varchar(255) NOT NULL')]`
+      s3_url (str): Full S3 URL containing the files to load into the destination table .
+      field_delimiter (str, optional): The character used to indicate how the fields in input files are delimited.
+              Defaults to `,`.
+      enclosed_by (str, optional): Single character string that specifies the fields enclosing character.
+              Defaults to empty string(no enclosing character).
+      escaped_by (str, optional): Single character string which indicates the escaping of delimiters and
+              other escape sequences. Defaults to backslack(`\\`),
+      overwrite (bool, optional): Whether to overwrite existing data in the destination Table. Defaults to `False`.
+    """
+
+    logger = get_logger()
+
+    user = aurora_credentials.get('user')
```

Comment:
How come you're using `get` here?

Reply:
You mean instead of `aurora_credentials['user']` ?

Reply:
Yeah, just curious why we would want `None` there instead of a KeyError if one of these is missing.